server/db/MockDB.go

```diff
@@ -129,6 +130,17 @@ func (db *mockDB) SavePlayer(player model.Player) error {
 	return nil
 }
 
+// SendMessage add a Message to a game chat.
+func (db *mockDB) AddMessage(id string, username string, message model.Message) (*model.Game, error) {
+	if game, ok := db.games[id]; ok {
+		fmt.Println("Adding a new Message")
```

Comment:
Hmmm... i realize we haven't been adding logs anywhere haven't we... ha ha. This probably is a decent thing to leave in. but doesn't add much data for debugging.

Reply:
Ah. this is the MockDB. probalby not too crucial.